### PR TITLE
Bug/claudia 4007. sdc url correction

### PR DESCRIPTION
**Pull request by hmunfru**
#### Reviewers
@jframos 

#### Description
Fix issue 4007. SDC url correction

#### Test
tested locally

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/sdc/manager/async/impl/TaskManagerImpl.java [hunk 1]
```diff
@@ -84,7 +85,8 @@ public Task updateTask(Task task) {
     @Override
     public Task load(Long id) throws EntityNotFoundException {
         Task task = taskDao.load(id);
-        task.setHref(MessageFormat.format(systemPropertiesProvider.getProperty(SDC_MANAGER_URL)+ Configuration.TASK_BASE_PATH, Long.valueOf(task.getId())
+        task.setHref(MessageFormat.format(systemPropertiesProvider.getProperty(SDC_MANAGER_URL)+
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/manager/async/impl/TaskManagerImpl.java]: blanck space before "+"

File: core/src/main/java/com/telefonica/euro_iaas/sdc/manager/async/impl/TaskManagerImpl.java [hunk 2]
```diff
@@ -73,8 +72,8 @@ public Task createTask(Task task) {
     @Override
     public Task updateTask(Task task) {
         task = taskDao.update(task);
-        task.setHref(MessageFormat.format(systemPropertiesProvider.getProperty(SDC_MANAGER_URL)+ Configuration.TASK_BASE_PATH, Long.valueOf(task.getId())
-                .toString(), task.getVdc()));
+        task.setHref(MessageFormat.format(systemPropertiesProvider.getProperty(SDC_MANAGER_URL)+
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/manager/async/impl/TaskManagerImpl.java]: It is the same in this line, leave a blanck space before "+"

Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/manager/async/impl/TaskManagerImpl.java]: NTC

**Comment by flopezag** [comment]
LGTM (pending coveralls...)

**Comment by hmunfru** [comment]
coverall https://coveralls.io/builds/2121079

**Comment by jframos** [comment]
LGTM

**Pull request merged**